src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,102 @@
+{
+  "properties": {
+    "textEditor.razor.advanced.formatOnType": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_FormattingOnTypeDisplayName%",
```

Comment:
You're using the resource ID format for JSON string resources, but you haven't supplied any JSON string resources.  Did you want to use the resource ID format for VS Package resources?

Reply:
Yea, this is the same id that will be coming from our resx file. Is there a different format on that? I didn't immediately see in the guide but definitely could have overlooked

Reply:
Ah, I found it in some usages in the devdiv repo. I think this is right ,assuming the GUID needed is the same as our `AboutDialogInfo` guid